reviews/2022-open-cluster-management-annual.md

```diff
@@ -0,0 +1,98 @@
+
+- Qiu Jian (Red Hat)	@qiujian16
+- Joshua Packer	(Red Hat)	@jnpacker	
+- David Eads	(Red Hat)	@deads2k	
+- Mike Ng	(Red Hat)	@mikeshng	
+- Min Kim	(Alibaba)	@yue9944882	
+- Yong Feng	(Alibaba)	@luckyfengyong
+
+## Project adoption
+
+OCM has 
+
+
+## Project goals
+
+OCM primary goal is to enable capabilities within the Kubernetes ecosystem to orchestrate themselves across multiple clusters and cloud providers. In the past year, we have achieved the following development outcomes.
+- Release cadence: 4 minor releases, roughly once every three months. The latest release is v0.9.0.
+- Key features added to the project (complete changelog can be found in [here](https://open-cluster-management.io/community/releases/)):
+  - 0.6
+    - Cluster Proxy add-on.
+    - Managed SrviceAccount add-on.
+    - Sync status of applied resources in ManifestWork API.
+    - Placement extensible scheduling.
+    - Helm chart interface for addon framework.
+  - 0.7
+    - New major features to Clusteradm CLI tool.
+      - Create, bind and view ManagedClusterSet.
+      - Create and view ManifestWork.
+      - Check the multicluster controlplane status.
+      - Upgrade hub and klusterlet.
+    - A default ManagedClusterSet
+    - New PolicySet API.
+  - 0.8
+    - ManifestWork update strategy.
+    - Global ClusterSet.
+    - Configuring feature gates for Klusterlet and Cluster Manager.
+    - Support host alaises for Klusterlet.
+    - Running policy addon using Clusteradm CLI tool.
+  - 0.9
+    - De-escalate ManifestWork agent privilege on Managed Clusters.
+    - Support referencing the add-on configuration with add-on related APIs.
+    - Allow targeting specific services within Managed Clusters.
+    - Application add-on is now able to expose custom Prometheus metrics via the Git subscription.
+
+In the near future, we aim to achieve the following technical goals:
+- Running multiple Hub on a single cluster.
+- Introduce ArgoCD pull model integration.
+- Supporting to communicate between hub and managed clusters in the proxy environment.
+- Add delete options to Managedcluster.
+- Add-on Framework enhancements.
+- A complete project roadmap can be found in [here](https://github.com/orgs/open-cluster-management-io/projects/2/views/9).
+
+We have built close connections with members in the CNCF organization for project integration. 
+We have attended multiple conferences such as KubeCon, DevConf and others.
+We are looking for more user adoptions and community collaborations constantly.
+
+## How the CNCF can help to achieve the upcoming goals
+
+- More chances to advocate the project.
+- More chances to collaborate with other projects in CNCF or even out of CNCF.
+- Technical writing support for project documents.
+
+## Incubation readiness
+
+- We are preparing for the incubation proposal.
```

Comment:
Are there more details that can be provided here? This doesn't feel very specific.